packages/playwright-ct-react/registerSource.mjs

```diff
@@ -44,7 +44,7 @@ export function pwRegister(components) {
  * @returns {component is JsxComponent}
  */
 function isComponent(component) {
-  return !(typeof component !== 'object' || Array.isArray(component));
+  return component.kind === 'jsx';
```

Comment:
This would result in false positives where a user object has a kind property with the jsx value. I was suggesting a property with a magic name in the issue comment. Btw, what do we do to the non-jsx notation in react? I think we still support it.

Reply:
> I was suggesting a property with a magic name in the issue comment

I've added the magic property. I still have to test everything thoroughly. There are still some bugs in there.

> Btw, what do we do to the non-jsx notation in react? I think we still support it

I also think this should be supported. I'll create a separate ticket for it.